Build/Build.cs

```diff
@@ -40,6 +40,9 @@ class Build : NukeBuild
 
     string PullRequestBase => GitHubActions?.BaseRef;
 
+    [Parameter]
+    readonly Configuration Configuration = IsLocalBuild ? Configuration.Debug : Configuration.CI;
```

Comment:
I think we would need to introduce targeted parameters to make this work.
E.g. Something along the lines of this.

```cs
[Parameter]
readonly Configuration Configuration = Configuration.Debug;

readonly Configuration BuildConfiguration = IsLocalBuild ? Configuration : Configuration.CI;
readonly Configuration TestConfiguration = IsLocalBuild ? Configuration : Configuration.Debug;
readonly Configuration PackConfiguration = IsLocalBuild ? Configuration : Configuration.Release;

```

Reply:
Why? The solution configurations `Debug` and `CI` already use the right project configurations.

Reply:
The solution configuration `CI` is (to the best of my knowledge) for building a subset of the projects in debug or release mode.
At least I'm not sure what `dotnet pack --configuration CI` would output.

Reply:
You're right. The "configuration" means different things depending on which `dotnet` command we pass. On the build-level, it's the solution config.